### PR TITLE
Fix typos in cmake README.md

### PR DESCRIPTION
**Pull request by leif81**

### CODE REVIEW

**Comment by leif81** [comment]
I signed it!

File: cmake/README.md [hunk 1]
```diff
@@ -123,15 +123,15 @@ It will generate *nmake* *Makefile* in current directory.
...
+     C:\Path\to\protobuf\cmake\build\solution>cmake -G "Visual Studio 12 2013 Win64" ^
      -DCMAKE_INSTALL_PREFIX=../../../../install ^
      ../..
 
-It will generate *Visual Studion* solution file *protobuf.sln* in current directory.
+It will generate *Visual Studio* solution file *protobuf.sln* in current directory.
 
 If the *gmock* directory does not exist, and you do not want to build protobuf unit tests,
-you need to add *cmake* command argument `-Dprotobuf_BUILD_TESTS=OFF` to disable testing.
+you need to add *cmake* command argument `-DBUILD_TESTING=OFF` to disable testing.
```
Comment (xfxyjwf) [inline cmake/README.md]: The flag name is changed to "protobuf_BUILD_TESTS" in https://github.com/google/protobuf/pull/783

Comment (leif81) [inline cmake/README.md]: Ah Ok, I wasn't aware of that other PR. These corrections were just for the `master` branch. I can certainly undo that fix, it'll just be a little misleading until the other PR gets merged.

**Comment by xfxyjwf** [comment]
Merging... Thanks for the patch!

**Pull request merged**